chapters/template-chapter/template-topic/slides/intro.md

```diff
@@ -13,9 +13,9 @@ They also contain a list as follows:
 
 ### Slides including images
 
-![CPU Internals](media/cpu-internals.jpg)
+![CPU Internals](../media/cpu-internals.jpg)
```

Comment:
There is a rendering issue here (file not found). I think `reveal-md` doesn't like the fact this image is in a different directory.

Reply:
I don't think there's a big problem if we duplicate/move the image to be under the `slides` directory for now. I know there might be reuse between slides and content, but we can open an issue and figure this out later.